docs/create_your_first_nn.md

```diff
@@ -0,0 +1,118 @@
+ input array into a one-dimensional array with 784 elements (28 * 28 = 784).
+ 
+Next, we've got a sequence of three **Dense()** layers. Dense layers are the most common layers in all kinds
+ of neural network architectures. They are also called fully connected layers because they connect every neuron 
+ in one layer to every neuron in the next layer.
+ 
+For each Dense layer we have specified its size. The first one has 300 neurons (or nodes), the second one has 100 neurons, 
+and the last one has 10 neurons (remember, we have 10 different classes to predict). 
+
+In each Dense layer we have also explicitly specified an activation function to be used. 
+Without activation functions a neural network would simply be passing data through linear nodes and would not be performing 
+any better than a simple linear regression model. Adding non-linear activation functions to the layers allows neural networks
+to capture data's complexity. There are different kinds of such functions. Here we've picked 
+[ReLu](https://en.wikipedia.org/wiki/Rectifier_(neural_networks)) as one of the most common choices for an activation function.    
+
+## Compiling a neural network 
+In the previous step we have defined the neural network's structure. In the next step, we need to decide *how* it will be trained - 
+what optimization algorithm will be used, what do we want to optimize for, and how do we evaluate progress. This is defined 
+in the *compile* step:
+
+```kotlin
+    model.compile(Adam(), Losses.SOFT_MAX_CROSS_ENTROPY_WITH_LOGITS, Metrics.ACCURACY)
+```
+
+* [Adam](https://arxiv.org/abs/1412.6980) is an optimization algorithm that our neural network will use to update the weights in its nodes as it trains.
+* Loss function is used to measure model's accuracy during the training. This is what you optimize for to improve your model's performance. 
+Here, the last layer of the model gives us 10 numbers - each representing a probability of the example belonging to each class. 
+Softmax Crossentropy with logits measures the probability error. 
+* Metrics allow you to monitor the training and evaluation of the model. Accuracy simply represents the number of correct 
+predicitons out of all predictions made.  
+
+At this point you can call `summary()` method to get a printout of the neural network's architecture. 
+
+```kotlin
+    model.summary()
+```
+
+```
+INFO  api.core.Sequential - =================================================================
+INFO  api.core.Sequential - Model: Sequential
+INFO  api.core.Sequential - _________________________________________________________________
+INFO  api.core.Sequential - Layer (type)                 Output Shape              Param #   
+INFO  api.core.Sequential - =================================================================
+INFO  api.core.Sequential - flatten_1(Flatten)           [784]                     0
```

Comment:
It will have alternative output in new artifact (output shapes will be changed)

Reply:
When will the new artefact be available so I could check that?